rdmo/accounts/tests/test_views.py

```diff
@@ -258,6 +291,36 @@ def test_remove_user_post(db, client):
         assert not get_user_model().objects.filter(username='user').exists()
 
 
+def test_remove_user_post_cancelled_when_update_is_true(db, client, settings):
+    settings.PROFILE_UPDATE = True
+    if settings.PROFILE_DELETE:
```

Comment:
yes think so too. In general, I would like to have the option to loop through these different test configs/combinations as well.. I can look into that but don't know if it will be too much for this PR.

Reply:
for now, I will keep it so that the settings are set to True/False at the start of each test case. We can keep in mind to refactor/improve the tests with that in another PR but also that the amount of test cases should be kept maintainable.

Reply:
I've added now a True/False option with  `@pytest.mark.parametrize('test_setting', boolean_toggle)` to most of the test cases. Then in each test case, only the relevant part is checked in a conditional eg:
```py
   if settings.PROFILE_UPDATE:
        assertTemplateUsed(response, 'profile/profile_update_form.html')
    else:
        assertTemplateUsed(response, 'profile/profile_update_closed.html')
```